src/logic/community/types.ts

```diff
@@ -185,6 +185,11 @@ export type BannedMemberProfile = BannedMember & {
 export type CommunityWithMembersCount = Community & {
   role: CommunityRole
   membersCount: number
+  voiceChatStatus: {
```

Comment:
We should add these properties to `Community` or create a new type to state that it contains data regarding voice chat. Mostly because this type is named `CommunityWithMembersCount` and these new properties are not related to that